parcel_creation.py

```diff
@@ -0,0 +1,42 @@
+import random
+import json
+
+PACKAGE_VOLUME_RANGE = 100
+
+
+def generating_parcel_number():
+    with open("parcel_list.json", "r") as f:
+        parcel_dict = json.load(f)
+
+    parcel_number = str(
+        random.choice(list(set([x for x in range(1, PACKAGE_VOLUME_RANGE)])))
+    )
+    while parcel_number in parcel_dict.keys():
```

Comment:
a na cholere taka kombinacja ? 
```suggestion
    parcel_number = len(parcel_dict.keys()) + 1
```